ledger/src/blockstore.rs

```diff
@@ -3316,8 +3296,14 @@ impl Blockstore {
         Ok(duplicate_slots_iterator.map(|(slot, _)| slot))
     }
 
-    pub fn last_root(&self) -> Slot {
```

Comment:
If there is concern about the removal of this `pub` function, I could re-add it and just make it call `Blockstore::max_root()`

Reply:
I'm not totally sure where blockstore.rs falls in our backward compatibility policy, but I think my preference is to preserve and deprecate `fn last_root()`